src/exts/tvdb_info/main.py

```
@@ -0,0 +1,126 @@
+                    value=str(i),
+                    description=result.overview[:100] if result.overview else None,
+                )
+                for i, result in enumerate(self.results)
+            ],
+        )
+        self.dropdown.callback = self._dropdown_callback
+        self.add_item(self.dropdown)
+        self.index = 0
+
+    def _get_embed(self) -> discord.Embed:
+        result = self.results[self.index]
+        original_title = result.name
+        en_title = result.translations["eng"]
+        original_overview = result.overview
+        en_overview = result.overviews["eng"]
+        if en_overview and en_overview != original_overview:
+            overview = f"{en_overview}"
+        elif not en_overview and original_overview:
+            overview = f"{original_overview}\n\n*No English overview available.*"
+        else:
+            overview = "*No overview available.*"
+        if original_title and original_title != en_title:
+            title = f"{original_title} ({en_title})"
+        else:
+            title = en_title
+        if result.id[0] == "m":
+            title = f"{MOVIE_EMOJI} {title}"
+            url = f"https://www.thetvdb.com/movies/{result.slug}"
+        else:
+            title = f"{SERIES_EMOJI} {title}"
+            url = f"https://www.thetvdb.com/series/{result.slug}"
+        embed = discord.Embed(title=title, color=discord.Color.blurple(), url=url)
+        embed.add_field(name="Overview", value=overview, inline=False)
+        embed.set_footer(text=THETVDB_COPYRIGHT_FOOTER, icon_url=THETVDB_LOGO)
+        embed.set_image(url=result.image_url)
+        return embed
+
+    async def _dropdown_callback(self, interaction: discord.Interaction) -> None:
+        if not self.dropdown.values or not isinstance(self.dropdown.values[0], str):  # noqa: PD011
+            log.error("Dropdown values are empty or not a string but callback was triggered.")
+            return
```

Hmm, should this really just return here? Maybe we can actually raise something so that the error handler will catch it and show it as an error even to the user, instead of it appearing as a silent failure on user's side. Same for the similar if below this